PiwikTracker/Queue.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  EventQueue.swift
+//  PiwikTracker
+//
+//  Created by Cornelius Horstmann on 09.10.16.
+//  Copyright © 2016 Mattias Levin. All rights reserved.
+//
+
+import Foundation
+
+/// An Element that is queueable.
+protocol Queueable {
```

Comment:
Not sure `Queueable` is a good description it has nothing to do with queuing. More about archiving or storing.